docs/en/samples/fuse_clean_policy.md

```diff
@@ -0,0 +1,110 @@
+# Demo - Set FUSE clean policy
+
+FUSE clean policy is set under the property `spec.fuse.cleanPolicy` of `Runtime`. There are 2 clean policies for FUSE Pod. `OnRuntimeDeleted` means that the FUSE Pod is cleaned only when the cache runtime is deleted, and `OnDemand` means that when the FUSE Pod is not used by any application Pod, the FUSE Pod will be cleaned.
+By default, the cleanup policy for FUSE Pod is `OnRuntimeDeleted` 
+
+## Prerequisites
+
+Before running this example, please refer to the [Installation Documentation](https://github.com/fluid-cloudnative/fluid/blob/master/docs/en/userguide/install.md) to complete the installation, and check that each Fluid component is normal run:
+
+```shell
+$ kubectl get pod -n fluid-system
+NAME                                        READY   STATUS    RESTARTS        AGE
+alluxioruntime-controller-86ddc878d-pc6g5   1/1     Running   7 (2m19s ago)   24h
+csi-nodeplugin-fluid-ccbk8                  2/2     Running   6 (2m19s ago)   24h
+dataset-controller-67bcb77b89-6xw7p         1/1     Running   4 (2m16s ago)   24h
+fluid-webhook-648ccc89c6-bq5rd              1/1     Running   4 (2m18s ago)   24h
+```
```

Comment:
Please check the reason of restarting. And I hope the count of restart times is 0. And in the latest version of fluid, alluxioruntime-controller will not be created by default.

Reply:
The reason is that I ran Fluid on single-node Kubernetes with my laptop. They would restart when I restarted Kubernetes. 
I've ran the latest version of Fluid on ACK, and everything seemed normal.